packages/rocketchat-personal-access-tokens/server/methods/generateToken.js

```diff
@@ -0,0 +1,31 @@
+Meteor.methods({
+	'personalAccessTokens:generateToken'({ tokenName }) {
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('not-authorized', 'Not Authorized', { method: 'personalAccessTokens:generateToken'});
+		}
+		if (!RocketChat.settings.get('API_Enable_Personal_Access_Tokens')) {
+			throw new Meteor.Error('not-authorized', 'Not Authorized', { method: 'personalAccessTokens:generateToken'});
```

Comment:
the error description here could say `Personal Access Tokens are currently disabled`. what do do you think?